src/components/AppProvider/utilities/createPolarisContext/createPolarisContext.ts

```diff
@@ -2,45 +2,42 @@ import {PolarisContext} from '../../../types';
 import {
   createThemeContext,
   ThemeProviderContextType as CreateThemeContext,
-} from '../../../ThemeProvider';
+} from '../../../../utilities/theme-provider';
 import {AppProviderProps} from '../../types';
 import StickyManager from '../StickyManager';
 import createAppProviderContext, {
   CreateAppProviderContext,
 } from '../createAppProviderContext';
+import Intl, {TranslationDictionary} from '../../../../utilities/intl';
+import ScrollLockManager from '../../../../utilities/scroll-lock-manager';
 
 export interface CreatePolarisContext extends AppProviderProps {
   stickyManager?: StickyManager;
 }
 
-export function createPolarisContext(): PolarisContext;
```

Comment:
This type was originally overloaded to prevent a breaking change, we should add a deprecation message